sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientSideRequestStatistics.java

```diff
@@ -304,11 +309,13 @@ public void serialize(
 
             try {
                 SystemInformation systemInformation = new SystemInformation();
-                long totalMemory = Runtime.getRuntime().totalMemory() / 1024;
-                long freeMemory = Runtime.getRuntime().freeMemory() / 1024;
-                long maxMemory = Runtime.getRuntime().maxMemory() / 1024;
+                Runtime runtime = Runtime.getRuntime();
```

Comment:
nit: redundant variable, Runtime.getRuntime() always return same static value

Reply:
Starihgt port from Spark30 feature branch - and this approach should still be slightly more efficient (because of only 1 instead of 4 method invocations and allocations on the stack) - to be honest I wouldn't have made the change for efficiency - doesn't really matter because it isn't called often enough - but for better readability.